com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/Includes/LegacySurfaceVertex.hlsl

```diff
@@ -0,0 +1,26 @@
+#ifndef UNITY_LEGACY_SURFACE_VERTEX
```

Comment:
Put this in it's own file instead of in the template so the generated code was a little cleaner